rs/nervous_system/clients/src/management_canister_client/tests.rs

```diff
@@ -119,6 +119,7 @@ async fn test_limit_outstanding_calls() {
         status: CanisterStatusType::Running,
         reserved_cycles: zero.clone(),
     };
+    std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
Do we need this?

Reply:
No, not sure how that got here, removed it